utils/webassembly/build-toolchain.sh

```diff
@@ -73,7 +74,7 @@ sed -i -e "s@\".*/include@\"../../../../share/wasi-sysroot/include@g" $TMP_DIR/$
 # Copy nightly-toolchain's host environment stdlib into toolchain
 
 if [[ "$(uname)" == "Linux" ]]; then
-  cp -r $NIGHTLY_TOOLCHAIN/usr/lib/swift/linux $TMP_DIR/$TOOLCHAIN_NAME/usr/lib/swift
+  cp -a $NIGHTLY_TOOLCHAIN/usr/lib/* $TMP_DIR/$TOOLCHAIN_NAME/usr/lib || true
```

Comment:
I'm not sure why need `|| true` :thinking:

Reply:
There's a single file that fails to get copied, related to clang I think. It doesn't cause problems as far as I understand, and it's a pain to test and to exclude just a single file. I can add a comment about that in the script. Not sure I'll be able to get the exact file name quickly, seems like GitHub erased logs for the previous actions in this PR (maybe because of the force push?), and I don't have access to powerful hardware to re-run builds quickly from scratch.

Reply:
OK, I'll try it on my local machine 👍

Reply:
I got the error. It seems that our toolchain's `lib/swift/clang` is a directory but nightly toolchain's one is a symlink.
```
cp: cannot overwrite directory '/tmp/tmp.wtHPlnPXwh/swift-wasm-DEVELOPMENT-SNAPSHOT-2020-05-06-a/usr/lib/swift/clang' with non-directory
```

So I added `clang-resource-dir-symlink` component to install to create that symlink